### DISCUSSION TITLE
How to extract the attribute lineage from the json generated from dispacher?

### DISCUSSION THREAD

**Discussion by julianzlzhang** [AbsaOSS/spline-spark-agent#385] [Q&A] [answered]
I plan to integrate spine spark agent into our existing system, I have implemented the dispacher which could send the json to our system .
But the json is complicated and I only want to extract the final attribute lineage (column lineage/field lineage) which looks like:
T1.C1 -> TA.C1
T2.C1 -> TA.C1
T3.C1 -> TB.C1
Is there any sample code I can refer to?

**Comment by wajda** [comment, marked as answer]
https://opensource.bigusdatus.com/spline/rest/docs/producer.html#operation/executionPlanUsingPOST

The property `attribute.childRefs` contains references to expressions/attributes that the given attribute was created from. It forms a DAG structure.

**Comment by julianzlzhang** [comment]
Thanks. now we can achieve it by a small amount code.
